spec/features/home_spec.rb

```diff
@@ -7,15 +7,6 @@
       Full stack web developer
     HEADLINE
 
-    # iff using Percy, make sure that devicon font has loaded, to keep screenshots consistent
-    if ENV.fetch('PERCY_BUILD_ID', nil).present?
-      wait_for do
-        page.evaluate_script('document.fonts.check("65px devicon")')
-      end.to eq(true)
```

Comment:
Hopefully Percy will be reliable with these SVGs?, but I'm not confident that it will be.